res/values-pl/strings.xml

```diff
@@ -1211,9 +1213,9 @@ Komendy Admina:
 Jeśli nie chcesz się logować, możesz nadal grać w trybie jednego gracza."
     </string>
     <string name="_2x_skin_resolution">"2x Rozdzielczość Skórki"</string>
-    <string name="pm">PM</string>
-    <string name="You_are_already_friends_">Jesteście już przyjaciółmi.</string>
-    <string name="Account_must_be_1_day_old_to_send_mail_">Account must be 1 day old to send mail.</string>
+    <string name="pm">pm</string>
```

Comment:
Please leave this as uppercase, this one isn't supposed to not be capitalized.

Reply:
Done! Please check under.